pom.xml

```diff
@@ -11,11 +11,21 @@
         <module>log4j-ecs-layout</module>
         <module>log4j2-ecs-layout</module>
         <module>logback-ecs-encoder</module>
-        <module>jul-ecs-formatter</module>
         <module>jboss-logmanager-ecs-formatter</module>
         <module>log4j2-legacy-tests</module>
         <module>log4j-legacy-tests</module>
         <module>logback-legacy-tests</module>
+        <module>jul-ecs-formatter</module>
+        <!-- IMPORTANT:
```

Comment:
The result will be that the release is not published on maven-central: It won't be promoted from staging.

This will be detected by #236 , so it is easy to simply abort the release, fix the module order and restart the release workflow.